prow/flagutil/config/config.go

```diff
@@ -43,6 +43,8 @@ type ConfigOptions struct {
 	// Inrepoconfig related flags
 	InRepoConfigCacheSize    int
 	InRepoConfigCacheDirBase string
+	// Controller runtime manager
+	TimeoutListingProwJobs int
```

Comment:
This is not related to configuration at all. Please add a new flag bundle under flagutil rather than reusing `ConfigOptions` even if it is just the one flag we're adding.